examples/ipc/server.py

```diff
@@ -0,0 +1,16 @@
+from discord import Intents
+from discord.ext import commands, ipc
+
+bot = commands.Bot(command_prefix="$", intents=Intents.all())
+server = ipc.Server("localhost", "my_secret_token", bot)
+
+
+@server.route()
+async def get_member_count(data):
+    guild = bot.get_guild(data.guild_id)  # get the guild.
+
+    return guild.member_count  # send the member count.
+
+
+server.run()
```

Comment:
This doesn't make sense. Why is there a blocking call to run the server that when it stops executing, starts to run the actual bot?